prediction-applier/src/main/resources/application.yml

```diff
@@ -12,6 +12,10 @@ hyland-experience:
   insight:
     prediction:
       endpoint: aws2-sqs://arn:aws:sqs:us-east-1:000000000000:hxinsight-prediction-queue
+    prediction-listener:
+      predictions-endpoint: http:localhost:8080/predictions?pageSize=10
+      prediction-processor-trigger-endpoint: quartz:prediction-processor-trigger?cron=0+*/1+*+?+*+*
```

Comment:
I don't think we want to make this configurable. I think we should only expose the polling period. (Also I think the default should be every 5 minutes, but I think this is every 1 minute?)

Reply:
okay, I'll try to somehow hide it but I have to make it configurable so we'll be able to test it

Reply:
done🤔 could you check if you like it?